Gems/Multiplayer/Code/Include/Multiplayer/Components/MultiplayerComponent.h

```diff
@@ -67,6 +67,10 @@ namespace Multiplayer
         NetworkEntityHandle GetEntityHandle();
         void MarkDirty();
 
+        //! Invoked when the NetworkEntity is attached and has RPCs bound via m_networkActivatedHandler
+        //! if m_networkActivatedHandler is registered via NetBindComponent::AddNetworkActivatedEventHandler 
+        virtual void OnNetworkActivated(){};
```

Comment:
IHMO It would be better to write this in a less technical framing. Why do I as a MultiplayerComponent author care about this event? What can I do once this event is received?

Reply:
I wanted to be technical here so users can understand why overriding OnNetworkActivated by itself is not sufficient. I can add a little more description here.